.gitlab-ci.yml

```diff
@@ -42,10 +45,13 @@ Nginx:
   - apk update && apk add git
   - git clone --depth 1 https://github.com/Ensembl/ensembl-2020-static-assests.git
   - git clone --depth 1 https://gitlab.ebi.ac.uk/kamal/ensembl-client-nginx.git
-  - docker build -t ${CONTAINER_IMAGE} -f ensembl-client-nginx/Dockerfile --no-cache .
+  - docker build -t ${CONTAINER_STAGE_IMAGE} -f ensembl-client-nginx/Dockerfile --no-cache .
+  - docker build -t ${CONTAINER_PROD_IMAGE} -f ensembl-client-nginx/Dockerfile.prod --no-cache .
   - echo "$GITLAB_REGISTRY_TOKEN" | docker login -u "$GITLAB_REGISTRY_USER" --password-stdin https://"$GITLAB_REGISTRY_URL"
-  - docker push ${CONTAINER_IMAGE}
-  - docker rmi ${CONTAINER_IMAGE}
+  - docker push ${CONTAINER_STAGE_IMAGE}
+  - docker push ${CONTAINER_PROD_IMAGE}
```

Comment:
Why would a push to staging cause a production image to be built at well?

Reply:
The way nginx redirects requests to backend API is through local.conf (nginx) file which is embedded inside the nginx image during build time. At the moment we need seprate nginx (local.conf). Its not good solution but it will change once we have ingress controller in place in kubernetes cluster.